source/mobile/mobile-overview.rst

```diff
@@ -1,37 +1,85 @@
+- Determine the ideal mobile deployment model for your organization.
+- Understand what’s required to build your own Mattermost Mobile Apps.
+- Deploy Mattermost Mobile Apps.
 
-This option is recommended for:
+.. note::
 
-1. Testing out the mobile applications
-2. Team Edition servers using no push notifications, or push notifications from Mattermost's :ref:`TPNS <tpns>` (Test Push Notification Service)
-3. Enterprise Edition servers using push notifications from Mattermost's :doc:`HPNS <mobile-hpns>` (Hosted Push Notification Service)
+    - Mattermost Cloud customers must use Mattermost pre-built public apps for mobile deployment. 
+    - Some features described in this guide are available only in Mattermost Professional or Mattermost Enterprise.
 
-Build the apps yourself
-------------------------
+Decision Summary
+----------------
 
-To customize the mobile apps, you can fork the open source repository and :doc:`compile your own versions <mobile-compile-yourself>` of the Mattermost mobile applications and Mattermost push proxy server. 
+When planning for a Mobile app deployment, you have two important decisions to make: 
 
-This option is recommended for:
+- What app will you deploy?
+- How will you deploy it?
 
-1. Organizations that want to customize the mobile apps
-2. Team Edition and Enterprise Edition servers that prefer to host their own push proxy server instead of using one of Mattermost's hosted versions
+Using Mattermost’s pre-built apps is the easiest and fastest approach since your users can download the apps from public app stores, including the `Apple App Store <https://www.apple.com/ca/app-store>`__ and the `Google Play Store <https://play.google.com/store>`__. Alternatively, you can use an EMM provider to enforce or restrict specific security policies. See `Using Mattermost’s Pre-Built Apps <https://docs.mattermost.com/mobile/use-prebuilt-mobile-apps.html>`__ to learn more about using Mattermost pre-built apps.
 
-Use an EMM provider with Managed App Configuration
----------------------------------------------------
+If you want to control the look, feel, or capabilities of the mobile apps, host your own push proxy server, or to maintain full control over app distribution, you can build your own mobile apps and manage your own app distribution. See `Building and Distributing Your Own Custom Mattermost Mobile Apps <https://docs.mattermost.com/mobile/build-custom-mobile-apps.html>`__ to learn more about working with custom apps.
 
-To meet other security and compliance policies, you can :doc:`deploy the Mattermost apps with any EMM provider <mobile-appconfig>` that supports `AppConfig <https://www.appconfig.org/members/>`__, such as MobileIron, Blackberry UEM, or Airwatch. You will still need to choose either the App Store/Google Play versions or apps compiled yourself to use in conjunction with the EMM provider.
+The following table summarizes the key differences between these two approaches:
```

Comment:
I can't see the table in the preview mode in GitHub. Is it displaying for you locally?

Reply:
Thanks all! Table should now be visible.